astro/src/pages/index.astro

```diff
@@ -48,35 +50,36 @@ const builder = sanityImageUrl(banner.image)
       </svg>
     </span>
 
-    <div class="h-[40vh] md:h-[100vh]"></div>
-    <div class="bg-neutral-900">
-      <!-- Upcoming Shows -->
-      <section id="shows" class="scroll-m-48 lg:pt-16">
-        <UpcomingShows />
-      </section>
-
-      <!-- Music -->
-      <section
-        id="music"
-        class="mt-20 opacity-0 transition duration-1000 ease-in-out"
-      >
-        <h2 class="p-4 text-3xl">MUSIC</h2>
-        <Music />
-      </section>
-
-      <!-- Video -->
-      <section
-        id="video"
-        class="opacity-0 transition duration-1000 ease-in-out"
-      >
-        <h2 class="p-4 text-3xl">VIDEO</h2>
-        <RandomSnacksVid snacksVids={snacksVids} client:idle />
-      </section>
-
-      <!-- Band info -->
-      <section id="band">
-        <BandInfo />
-      </section>
+    <div class="bg-none pt-[30vh] sm:pt-[100vh]">
```

Comment:
This diff got a little messy due to formatting changes, so it's kind of hard to tell at a glance what changed. 

Previously we had a `100vh` div with no background that stacked on top of the page content to expose the `position: fixed` hero image. That was causing some weird page scroll stickiness here and there, so I opted to take that out and add `100vh` padding on top of the page content, and add a container inside of that with the black background